easy_infra.yml

```diff
@@ -95,28 +95,28 @@ packages:
   fluent-bit:
     helper:
     - all
-    version: v2.1.10
+    version: v2.2.0
     version_argument: --version
   git:
     helper:
     - all
     version: v2.42.0
     version_argument: --version
   kics:
-    version: v1.7.8
+    version: v1.7.10
```

Comment:
We need to keep this at 10 and not 11 until the upstream irons out the multiplatform builds per https://github.com/Checkmarx/kics/issues/6800